packages/salesforcedx-sobjects-faux-generator/src/generator/fauxClassGenerator.ts

```diff
@@ -61,7 +50,16 @@ export class FauxClassGenerator {
     // TBD what are these mapped to and how to create them
     // ['calculated', 'xxx'],
     // ['masterrecord', 'xxx'],
-    ['complexvalue', 'Object']
+    ['complexvalue', 'Object'],
```

Comment:
How did you arrive at the extensive list of all the different types?

Reply:
Referencing: https://developer.salesforce.com/docs/atlas.en-us.api_meta.meta/api_meta/meta_field_types.htm#meta_type_fieldtype

Still need to handle:
- MetadataRelationship
- IndirectLookup
- Hierarchy
- File
- CustomDataType
- Html

Do you know what the status of these are?